addons/website/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" name="Team Basic">
+    <section class="s_company_team_basic o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10">
+                <div class="o_grid_item g-col-lg-12 g-height-2 col-lg-12" style="grid-area: 1 / 1 / 3 / 13; --grid-item-padding-y: 32px; --grid-item-padding-x: 32px; z-index: 1;">
+                    <h3 class="o_default_snippet_text" style="text-align: center;">Discover our executive team</h3>
+                </div>
+                <div class="o_grid_item g-col-6 g-col-lg-3 g-height-8 col-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 1 / 11 / 4; --grid-item-padding-y: 16px; --grid-item-padding-x: 32px; z-index: 2;">
+                    <p class="o_not_editable" contenteditable="false" style="text-align: center;">
+                        <img alt="" class="o_editable_media img-fluid rounded-circle" src="/web/image/website.s_company_team_image_1" style="width: 100%; text-align: center;"/>
```

Comment:
The editor add specifically `width: 100% !important;`, there must be a reason, so I would stick to that.
I think the `text-align: center;` can be removed actually

Reply:
@anso-odoo I don't know why too... :thinking: maybe @bvr-odoo has an answer to that?

Reply:
> @anso-odoo I don't know why too... 🤔 maybe @bvr-odoo has an answer to that?

I don't know, probably because the part of the code that checks if 'important' is necessary doesn't work correctly.
Maybe it was introduced by [this commit](https://github.com/odoo/odoo/commit/d3c3dab8950abc25b29937605091d8ce32305fa4)